codingtovictory/day6/dailyChallenge/day6DailyChallenge.html

```diff
@@ -0,0 +1,93 @@
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+<body>
+
+    <h1>CHECKERBOARD</h1>
+    <div class="container">
+        <div class="row">
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+        </div>
+        <div class="row">
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+        </div>
+        <div class="row">
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+        </div>
+        <div class="row">
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+        </div>
+        <div class="row">
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+        </div>
+        <div class="row">
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+        </div>
+
+
+    <a href="https://www.pexels.com/photo/city-hotel-building-office-15903178/"><img src="Pics/pictures" style="height: 500px;width: 500px;padding: 150px;"></a>
```

Comment:
Try to stay await from inline styling and move these properties to your style sheet. Totally ok to do this but best practice is to decouple. Keep HTML with HTML and CSS with CSS. @OnkurLal @Bs-Sec